script/Ciel/Day 9/09_01_CIEL09_4.txt

```diff
@@ -307,7 +307,7 @@ However cunning a Dead Apostle may be, she always uncovers its identity."
 -- Page 397, Offset 23156.
 -- 「成功率は実に１００パーセント。ほとんど奇蹟よこんなの。あの子が<鬼子|おにご>の扱いじゃなきゃ聖女認定されてるくらい。
 // Nasu just loves 鬼子, really means shunned / outcast here
-"She literally has a 100% success rate.
+"She literally has a hundred percent success rate.
```

Comment:
"a one hundred percent success rate"  sounds slightly more right to me here. More emphatic?